.circleci/compute_merge_commit.sh

```diff
@@ -22,6 +22,8 @@ err=0
 
 GREEN() { echo -e "\n\033[0;32m$1\033[0m"; }
 
+mkdir -p /tmp/workspace
```

Comment:
Can we update the documentation at the top of this file, and add comments above each top-level set of steps? (Okay to do in a follow up.)